cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDClaim;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+
+@Component
+public class ClaimMapper implements Mapper<CCDClaim, ClaimData> {
+
+    private final PersonalInjuryMapper personalInjuryMapper;
+    private final HousingDisrepairMapper housingDisrepairMapper;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "housingDisrepairMapper" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1068)